Focus/Prism.swift

```diff
@@ -6,7 +6,25 @@
 //  Copyright (c) 2015 TypeLift. All rights reserved.
 //
 
-/// A Prism is an `Iso` where one of the functions is partial.
+/// A `Prism` describes a way of focusing on potentially more than one target structure.  Like
+/// an `Iso` a `Prism` is invertible, but unlike an `Iso` multi-focus `Prism`s are incompatible with
+/// inversion in general.  Because of this a `Prism` describes a `Lens` that can branch depending on
```

Comment:
A `Prism` is not a `Lens` (unless it is also an `Iso`).  This sentence needs rewording.